azure-iot-device/azure/iot/device/iothub/models/pnp_translation.py

```diff
@@ -21,6 +22,20 @@ def method_request_to_command(method_request):
     # )
 
 
-def twin_patch_to_writable_property(twin_patch):
-    """Given a Twin Patch (dict), returns a WritableProperty"""
-    pass
+def twin_patch_to_client_property_collection(twin_patch):
+    """Given a Twin Patch (`dict`), returns a `ClientPropertyCollection` object"""
+    obj = ClientPropertyCollection()
+    obj.backing_object = twin_patch
+    return obj
+
+
+def twin_to_client_properties(twin):
+    """Given a `Twin` object, return a `ClientProperties` object"""
+    obj = ClientProperties()
+    obj.backing_object = twin.reported_properties
```

Comment:
does a ClientProperties have a `backing_object` attribute? It seems like it doesn't